cluster/images/etcd/.gitignore

```diff
@@ -0,0 +1,4 @@
+etcd
+etcdctl
+etcd-*-linux-amd64.tar.gz
+etcd-*-linux-amd64/
```

Comment:
The trailing slash?  That specifies that it only matches a directory but not a file. https://git-scm.com/docs/gitignore